src/main/java/val/Result.java

```diff
@@ -108,13 +108,6 @@ public static Result from(String message, String code) {
 	return new Result(message, code);
     }
 
-    /**
-     * Perform a value based comparison against `other`.
-     * Returns whether `other` is also a `Result` with the
-     * same attributes as this Result.
-     * @param other Object to test equality against.
-     * @return Whether `other` is equal to this.
-     */
```

Comment:
I added this before adjusting the way the API documentation was generated so it doesn't include undocumented methods...it doesn't seem necessary and will now look nice without it.